packages/material-ui/src/TableSortLabel/tableSortLableClasses.d.ts

```diff
@@ -0,0 +1,7 @@
+import { TableSortLabelClassKey } from './TableSortLabel';
```

Comment:
The name of the file is incorrect, should be `tableSortLabelClasses.d.ts`

Reply:
Embarrassing... lol Thank you 👍🏻